pom.xml

```diff
@@ -532,4 +528,4 @@ SOFTWARE.
       </build>
     </profile>
   </profiles>
-</project>
+</project>
```

Comment:
@baudoliver7 I suggest reverting this part, if xcop doesn't complain than xml formating is fine